jazzy/distribution.yaml

```diff
@@ -1143,6 +1143,26 @@ repositories:
       url: https://bitbucket.org/dataspeedinc/dataspeed_can.git
       version: ros2
     status: maintained
+  dbw_ros:
+    doc:
+      type: git
+      url: https://bitbucket.org/dataspeedinc/dbw_ros.git
+      version: ros2
+    release:
+      packages:
+      - ds_dbw
+      - ds_dbw_can
+      - ds_dbw_joystick_demo
+      - ds_dbw_msgs
```

Comment:
I'll point out that in [Humble](https://github.com/ros/rosdistro/blob/61153d766f3c0e75ac50dc8e058a8ebbd19bca4a/humble/distribution.yaml#L1542-L1580), this has a lot more packages.  That's not a problem, per-se, but is this what you intended?